### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by haesleinhuepf**
# Description

This is just adding some more tests. I would like to make sure the functions do what they are supposed to do

## Type of change

* [ ] Bug-fix
* [ ] New feature
* [ ] Breaking change
* [ ] Documentation update

## Tests

* [ ] I adapted existing tests, because
* [ ] I added new tests to cover the code change
* [ ] All tests pass with my change

## Final checks

* [ ] My change is the minimal possible work for the desired feature/fix
* [ ] I updated the documentation where necessary to cover the change

### CODE REVIEW

File: tests/test_match_labels.py [hunk 1]
```diff
@@ -15,3 +15,13 @@ def test_match_labels():
     
     assert(all(match))
 
+
+def test_match_labels_2():
+    labels_x = np.asarray([1, 1, 1, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
+    labels_y = np.asarray([1, 1, 2, 0, 0, 3, 3, 4, 4, 4, 0], dtype=np.uint8)
+
+    reference_y_matched = np.asarray([1, 1, 5, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
```
Comment (haesleinhuepf) [inline tests/test_match_labels.py:23]: Hi Johannes @jo-mueller 

could you please take a look at that test? I think this should fail. I think, the "5" should be a "1" in the reference. What do you think?

Comment (jo-mueller) [reply tests/test_match_labels.py:23]: Yes, `reference_y_matched ` should essentially be a copy of `labels_x ` here.

Comment (jo-mueller) [reply tests/test_match_labels.py:23]: A shorter version of this test could be:

```python
labels_x = np.asarray([1, 1, 1, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
labels_y = np.asarray([1, 1, 2, 0, 0, 3, 3, 4, 4, 4, 0], dtype=np.uint8)

assert np.array_equal(labels_x, bputils.label.match_labels(labels_x, labels_y))
```

Comment (jo-mueller) [reply tests/test_match_labels.py]: Yes, that's true. I changed that in fd0f669ea1ad8a6510c57f56998ae4f605dc1f9b. Now there are two tests for the label matching - is this necessary or does the 2nd test cover something I overlooked?

**Comment by jo-mueller** [comment]
I added another test script for /label/_label_overlap_matrix.py

File: tests/test_match_labels.py (hunk 1 above)
Comment (haesleinhuepf) [reply tests/test_match_labels.py:23]: Yes, no I got it! Thanks :-)

**Comment by haesleinhuepf** [comment]
Great! That's a good additional test.

**Pull request merged**